<launch>

    <include file="$(find prizm)/launch/twist_controller.launch" />
    <include file="$(find lidar2d)/launch/lidar2d.launch" />

    <!-- launch the node -->
    <node name="avoid_obstacle" pkg="birds_eye" type="avoid_obstacle" respawn="true" respawn_delay="10" output="screen">
        <!-- Use a specific laserscan path -->
        <param name="scan_topic" type="string" value="/lidar2d/detect/scan" /> 
    </node>
</launch>
